src/main/java/jdbc/LazyFetchRowMapper.java

```diff
@@ -34,7 +39,7 @@ protected void setAssociation(ResultSet resultSet, T instance) throws NoSuchFiel
             }
 
             final Field collectionField = clazz.getDeclaredField(association.getFieldName());
-            List proxy = createProxy(instance, association.getEntityClass());
+            List proxy = createProxy(instance, association.getAssociatedEntityClass());
             ReflectionFieldAccessUtils.accessAndSet(instance, collectionField, proxy);
```

Comment:
```suggestion
ReflectionFieldAccessUtils.accessAndSet(
                    instance,
                    clazz.getDeclaredField(association.getFieldName()),
                    createProxy(instance, association.getAssociatedEntityClass())
            );
```

변수에 담은후 별다른 사용을 하지 않고 있을때, 이는 불필요한 변수선언이 되었다고 볼수 있을것 같은데 어떻게 생각하시나요?

Reply:
아 넵넵, 가끔 변수를 뽑는 과정이 너무 복잡할 때 미리 선언해두고 사용하는 버릇이 있는데, 말씀하신 것 처럼 이정도 경우에서는 바로 전달하는 편이 좋을 수도 있을 것 같습니다.

그런데 이부분은 제가 지금 스캔해보니 전반적으로 이렇게 코드를 작성하는 버릇이 있는 것 같네요. 한번 보일때마다 판단해서 줄여보도록 하겠습니다 :)